### PR TITLE
[zh-cn]: sync translation for `File` and `File()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

https://developer.mozilla.org/en-US/docs/Web/API/File
https://developer.mozilla.org/en-US/docs/Web/API/File/File

### Related issues and pull requests

depends on https://github.com/mdn/content/pull/33663

part of the https://github.com/mdn/translated-content/issues/20165

### CODE REVIEW

File: files/zh-cn/web/api/file/index.md [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
 
-## 方法
+- {{DOMxRef("File.lastModified")}} {{ReadOnlyInline}}
+  - : 返回文件的最后修改时间，以 UNIX 纪元（1970 年 1 月 1 日午夜）以来的毫秒为单位。
+- {{DOMxRef("File.lastModifiedDate")}} {{Deprecated_Inline}} {{ReadOnlyInline}} {{Non-standard_Inline}}
+  - : 返回 `File` 对象引用的文件的最后修改时间的 {{JSxRef("Date")}}。
+- {{DOMxRef("File.name")}} {{ReadOnlyInline}}
+  - : 返回 `File` 对象引用的文件的名称。
+- {{DOMxRef("File.webkitRelativePath")}} {{ReadOnlyInline}}
+  - : 返回 {{DOMxRef("File")}} 对象相对于 URL 的路径。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/file/index.md:36]:
```suggestion
  - : 返回 `File` 对象相对于 URL 的路径。
```

File: files/zh-cn/web/api/file/index.md [hunk 2]
```diff
@@ -1,46 +1,43 @@
 ---
 title: File
 slug: Web/API/File
+l10n:
+  sourceCommit: 8fd2ee72038310e3ecc387df235ffac1cb08775c
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/file/index.md:5]:
```suggestion
  sourceCommit: 467508d83e320be0680f334c3455d3cc232bce42
```

File: files/zh-cn/web/api/file/file/index.md [hunk 3]
```diff
@@ -1,35 +1,42 @@
...
+  - : 表示文件名或文件路径的字符串。
+- `options` {{optional_inline}}
 
-  - : 选项对象，包含文件的可选属性。可用的选项如下：
+  - : 包含文件可选属性的选项对象。可用选项如下：
 
-    - `type`: {{domxref("DOMString")}}，表示将要放到文件中的内容的 MIME 类型。默认值为 `""` 。
-    - `lastModified`: 数值，表示文件最后修改时间的 Unix 时间戳（毫秒）。默认值为 {{jsxref("Date.now()")}}。
+    - `type` {{optional_inline}}
+      - : 表示将放入文件的内容的 MIME 类型的字符串。默认值为`""`。
```
Comment (yin1999) [inline files/zh-cn/web/api/file/file/index.md:30]:
```suggestion
      - : 表示将放入文件的内容的 MIME 类型的字符串。默认值为 `""`。
```

File: files/zh-cn/web/api/file/index.md [hunk 4]
```diff
@@ -1,46 +1,43 @@
...
 ---
 
-{{APIRef}}
+{{APIRef("File API")}}{{AvailableInWorkers}}
 
-文件（**`File`**）接口提供有关文件的信息，并允许网页中的 JavaScript 访问其内容。
+**`File`** 接口提供有关文件的信息，并允许网页中的 JavaScript 访问其内容。
 
-通常情况下， `File` 对象是来自用户在一个 {{HTMLElement("input")}} 元素上选择文件后返回的 {{domxref("FileList")}} 对象，也可以是来自由拖放操作生成的 {{domxref("DataTransfer")}} 对象，或者来自 {{domxref("HTMLCanvasElement")}} 上的 `mozGetAsFile`() API。在 Gecko 中，特权代码可以创建代表任何本地文件的 File 对象，而无需用户交互（有关详细信息，请参阅[注意事项](#注意事项)。
+`File` 对象通常从用户使用 {{HTMLElement("input")}} 元素选择文件而返回的 {{DOMxRef("FileList")}} 对象中检索，或者从拖放操作的 {{DOMxRef("DataTransfer")}} 对象中返回。
```
Comment (yin1999) [inline files/zh-cn/web/api/file/index.md:12]: 这句的翻译还可以改一下，后半句有点乖乖的

Comment (skyclouds2001) [inline files/zh-cn/web/api/file/index.md:12]:
```suggestion
`File` 对象通常从用户使用 {{HTMLElement("input")}} 元素选择文件返回的 {{DOMxRef("FileList")}} 对象中检索，或者从拖放操作返回的 {{DOMxRef("DataTransfer")}} 对象中检索。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**